sdk/mobile-center-crashes/src/test/java/com/microsoft/azure/mobile/crashes/UncaughtExceptionHandlerTest.java

```diff
@@ -11,6 +11,7 @@
 import com.microsoft.azure.mobile.utils.DeviceInfoHelper;
```

Comment:
Code analysis now reveals unused reports. We encourage using Android Studio commit dialog and checking the various options to refactor, check, clean imports etc...